src/locales/de/battle.json

```
@@ -32,6 +32,7 @@
   "attackMissed": "Die Attacke hat {{pokemonNameWithAffix}} verfehlt!",
   "attackHitsCount": "{{count}}-mal getroffen!",
   "rewardGain": "Du erhältst {{modifierName}}!",
+  "rewardGainCount": "Du erhältst {{count}} {{modifierName}}!",
```

```suggestion
  "rewardGainCount": "Du erhältst {{count}} $t(modifierName, {\"count\": {{count}} })!",
```

This is necessary so plural can be handled in the future.
e.g. instead of `1 Beeren` you can then control is to show `1 Beere`, `2 Beeren` etc.